net/sockstats/sockstats_tsgo.go

```diff
@@ -276,3 +286,73 @@ func isLikelyCellularInterface(ifName string) bool {
 		strings.HasPrefix(ifName, "ww") || // systemd naming scheme for WWAN
 		strings.HasPrefix(ifName, "pdp") // iOS
 }
+
+// radioMonitor tracks usage of the cellular radio, approximates the power state transitions,
+// and reports the percentage of time the radio was on.
+type radioMonitor struct {
+	// usage tracks the last time (as unix timestamp) the radio was used over the last hour.
+	// Values are indexed by the number of seconds since the beginning of the current hour.
+	usage [3600]int64
```

Comment:
May want to add a `const` for the 3600 since it shows up in a bunch of places.

Reply:
done